apps/condo/domains/news/tasks/publishSharedNewsItemsByNewsItem.js

```diff
@@ -0,0 +1,20 @@
+const { find, getById } = require('@open-condo/keystone/schema')
+const { createTask } = require('@open-condo/keystone/tasks')
+
+const { publishSharedNewsItem } = require('@condo/domains/news/tasks')
+
+/**
+ * @param {string} newsItemId
+ * @param {Array} sharedNewsItemsIds
+ * @returns {Promise<void>}
+ */
+async function publishSharedNewsItemsByNewsItem (newsItemId, sharedNewsItemsIds) {
```

Comment:
I get what did you mean, but IMO it would be better to simplify name of this function to 
```suggestion
async function publishSharedNewsItemsBy (newsItemId, sharedNewsItemsIds) {
```